Source/Core/VideoBackends/Vulkan/VKPerfQuery.h

```diff
@@ -40,7 +40,7 @@ class PerfQuery : public PerfQueryBase
   struct ActiveQuery
   {
     u64 fence_counter;
-    PerfQueryType query_type;
+    PerfQueryGroup query_type;
```

Comment:
I don't feel strongly about bloating out this PR, so it could be done later, but I think this (and `EnableQuery` / `DisableQuery`) should call this 'group' instead of type to better describe what is used.  Especially since there is already a 'type' enum.

Maybe just rename this for now?

Reply:
All other backends call it query_type too.

Reply:
Ok, like I said, fine to just change the ones touched and we can cleanup the other references later

Reply:
Done.